spectral.txt

```diff
@@ -0,0 +1,93 @@
+function Coh = SCoh_W(y,x,alpha,nfft,Noverlap,Window,opt,P)     
+% Coh = SCoh_W(y,x,alpha,nfft,Noverlap,Window,opt,P)
+% Welch's estimate of the Cyclic Spectral Coherence of signals y and x 
```

Comment:
It says Welch's estimation here and apparently Welch's method is one of the ways to create an estimation.